frontend/src/components/Navbar.js

```diff
@@ -0,0 +1,101 @@
+import React, { useState } from 'react'
+import styled from 'styled-components'
+
+const Navbar = () => {
+    const [isOpen, setIsOpen] = useState(false);
+
+    return (
+        <Nav>
+            <Logo href="">
+                L<span>OGO</span>
+            </Logo>
+            <MenuIcon onClick={() => setIsOpen(!isOpen)}>
+                <span />
+                <span />
+                <span />
+            </MenuIcon>
+            <Menu isOpen={isOpen}>
+                <MenuLink href="">Bebidas</MenuLink>
+                <MenuLink href="">Kitss</MenuLink>
+                <MenuLink href="">Copos e taças</MenuLink>
+            </Menu>
+        </Nav>
+    )
+}
+
+const Nav = styled.div`
+    height: 80px;
+    padding: 0 2rem;
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+    flex-wrap: wrap;
+    background: white;
+`;
+
+const MenuIcon = styled.div`
+    display: none;
+    flex-direction: column;
+    cursor: pointer;
+
+    span {
+        height: 2px;
+        width: 25px;
+        background-color: gray;
+        margin-bottom: 4px;
+        border-radius: 2px;
+    }
+
+    @media (max-width: 768px){
+        display: flex;
+    }
+`;
+
+const MenuLink = styled.a`
+    padding: 1rem 2rem;
+    cursor: pointer;
+    text-align: center;
+    text-decoration: none; 
+    color: #67bc96;
+    transition: all 0.3s ease-in;
+    font-size: 0.9rem;
+
+    &:hover {
+       color: #7b7fda;
+    }
+`;
+
+const Menu = styled.div`
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    position: relative;
```

Comment:
Acredito que falte uma cor de fundo no Header mobile, olha como está:

 ![screencapture-localhost-3000-2021-11-05-12_18_05](https://user-images.githubusercontent.com/54550926/140535237-a370a9f9-4e1f-4158-ac46-fcdf39e4286d.png)

Acredito que adicionando uma cor aqui, já da uma melhorada:
```suggestion
    position: relative;
    background: white;
```